openlibrary/macros/daisy.html

```diff
@@ -1,19 +1,18 @@
-$def with(page, url='', msg='', block_name=None)
+$def with(page, url='', enc=False, block_name=None)
```

Comment:
Give this a longer name. Right now my only guess is `encoding`? Which doesn't make sense here.

Reply:
Let's call this `protected`